src/lib/contexts/userContext.js

```diff
@@ -0,0 +1,40 @@
+import React, { createContext, useCallback, useRef } from 'react'
+import { defaultAccountValue } from '../constants/user'
+
+export const UserContext = createContext()
+
+export const UserContextProvider = props => {
+  const isLoggedIn = useRef(false)
```

Comment:
1. use single variable keeping all context vars
2. it should be a state vars so we would be able to use effects on the vars exported from context